pkg/controller/import-controller_test.go

```diff
@@ -409,11 +409,152 @@ var _ = Describe("Import test env", func() {
+		result := reconciler.getSecretName(pvcNoAnno)
+		Expect(result).To(Equal(""))
+	})
+})
+
+var _ = Describe("getCertConfigMap", func() {
+	testConfigMap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "configMapName",
+			Namespace: "default",
+		},
+	}
+
+	It("should find the configmap if PVC has one defined, and it exists", func() {
+		pvcWithAnno := createPvc("testPVCWithAnno", "default", map[string]string{AnnCertConfigMap: "configMapName"}, nil)
+		reconciler := createImportReconciler(pvcWithAnno, testConfigMap)
+		cm, err := reconciler.getCertConfigMap(pvcWithAnno)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(cm).To(Equal(testConfigMap.Name))
+	})
+
+	It("should return the configmap name if PVC has one defined, but doesn't exist", func() {
+		pvcWithAnno := createPvc("testPVCWithAnno", "default", map[string]string{AnnCertConfigMap: "doesnotexist"}, nil)
+		reconciler := createImportReconciler(pvcWithAnno)
+		cm, err := reconciler.getCertConfigMap(pvcWithAnno)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(cm).To(Equal("doesnotexist"))
+	})
+})
+
+var _ = Describe("getInsecureTLS", func() {
+	configMapName := "cdi-insecure-registries"
+	host := "myregistry"
+	endpointNoPort := "docker://" + host
+	hostWithPort := host + ":5000"
+	endpointWithPort := "docker://" + hostWithPort
+
+	table.DescribeTable("should", func(endpoint string, configMapExists bool, insecureHost string, isInsecure bool) {
+		var reconciler *ImportReconciler
+		pvc := createPvc("testPVC", "default", map[string]string{AnnEndpoint: endpoint}, nil)
+		if configMapExists {
+			cm := &corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      configMapName,
+					Namespace: "cdi",
+				},
+			}
+
+			if insecureHost != "" {
+				cm.Data = map[string]string{
+					"test-registry": insecureHost,
+				}
+			}
+			reconciler = createImportReconciler(pvc, cm)
+		} else {
+			reconciler = createImportReconciler(pvc)
+		}
+		result, err := reconciler.isInsecureTLS(pvc)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(result).To(Equal(isInsecure))
+	},
+		table.Entry("return true on endpoint with no port, and cdi configmap exists, and host defined", endpointNoPort, true, host, true),
+		table.Entry("return true on endpoint with port, and cdi configmap exists, and host with port", endpointWithPort, true, hostWithPort, true),
+		table.Entry("return false on endpoint with no port, and cdi configmap exists, and host with port", endpointNoPort, true, hostWithPort, false),
+		table.Entry("return false on endpoint with port, and cdi configmap exists, and host defined", endpointWithPort, true, host, false),
+		table.Entry("return false on endpoint with no port, and no cdi configmap exists, and blank host", endpointNoPort, false, "", false),
+		table.Entry("return false on blank endpoint, and cdi configmap exists, and host defined", "", true, host, false),
+	)
+})
+
+var _ = Describe("getInsecureTLS", func() {
```

Comment:
Do you mean getContentType?
(this test doesn't do anything TLS-related)

Reply:
I did mean that, copy and paste strikes again.